src/mixins/dpValidateMixin.js

```diff
@@ -45,14 +45,14 @@ export default {
         customErrors.forEach(error => dplan.notify.notify('error', Translator.trans(error)))
 
         if (customErrors.length === 0) {
-          const nonEmptyFieldNames = [...new Set(
-            invalidFields
-              .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
-              .filter(Boolean)
-          )]
+          const nonEmptyFieldNames = invalidFields
+            .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
+            .filter(Boolean)
 
-          if (nonEmptyFieldNames.length) {
-            const fieldsString = nonEmptyFieldNames ? nonEmptyFieldNames.join(', ') : ' '
+          const nonEmptyUniqueFieldNames = nonEmptyFieldNames.filter((field, idx) => nonEmptyFieldNames.indexOf(field) === idx)
```

Comment:
```suggestion
          const nonEmptyUniqueFieldNames = nonEmptyFieldNames.filter((field, idx, arr) => arr.indexOf(field) === idx)
```

Reply:
and I think with tht, You concat it again with the upper Block.

Reply:
done https://github.com/demos-europe/demosplan-ui/pull/634/commits/8847423146fad341075cdadde131bd52b7763110